spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java

```diff
@@ -326,6 +326,50 @@ public void testFilesUnpartitionedTable() throws Exception {
     TestHelpers.assertEqualsSafe(filesTable.schema().asStruct(), expected.get(0), actual.get(0));
   }
 
+  @Test
+  public void testAggregatedMetadataTableInStage() throws Exception {
```

Comment:
This test name is incorrect. Can you please update it to `testAllMetadataTablesWithStagedCommits()`?

Reply:
OK